.travis.yml

```diff
@@ -37,6 +37,10 @@ env:
     - VERSION="13.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
 
 install:
+  #  FIXME:
+  - git clone --branch 13.0-mig_account_fiscal_position_rule_tbi
+    https://github.com/acsone/account-fiscal-rule.git PR100
+  - rm -rf account_fiscal_position_rule && cp -ar PR100/account_fiscal_position_rule .
```

Comment:
```suggestion
```

Reply:
BTW, it's best to not do these changes and just order a TravisCI rebuild once the dependencies are merged. It you went that way, I would have merged this right away.

Reply:
Done, thanks